src/components/CalenderEntry.tsx

```diff
@@ -1,40 +1,86 @@
 import React from 'react';
-import { Text, View } from 'react-native';
-import { style } from '../util/Style';
+import {Text, View} from 'react-native';
+import {style} from '../util/Style';
+import {Lecture} from "../Service/networking-service";
 
 interface CalendarEntryProps {
-  name: string;
-  room: string;
-  date: string;
-  type: 'PRESENCE' | 'ONLINE';
-  startTime: string;
-  endTime: string;
+    name: string;
+    room: string;
+    date: string;
+    type: 'PRESENCE' | 'ONLINE';
+    startTime: string;
+    endTime: string;
 }
 
-function formatStartEndTime(startTime: string, endTime: string): string {
-  let newStartTime = new Date(startTime)
-    .toTimeString()
-    .split(' ')[0]
-    .slice(0, 5);
-  let newEndTime = new Date(endTime).toTimeString().split(' ')[0].slice(0, 5);
+function formatStartEndTime(startTime: Date, endTime: Date): string {
```

Comment:
Das scheint mir sehr unverständlich, wieso hast du das verändert und was tut das?

Reply:
weil ich das quasi das in einem kurzen punkt brauche und es schneller ging für mich

Reply:
Ahh okay